#! /bin/sh -e
## 315_chage.1_document_expiration_removal.dpatch by Federico Grau <grauf@rfa.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Document expiraiton removal in chage(1)
## DP: Debian bug #304542

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@

--- shadow-4.0.3.ori/man/chage.1	2005-04-13 14:45:33.000000000 -0400
+++ shadow-4.0.3/man/chage.1	2005-04-13 15:01:27.000000000 -0400
@@ -71,6 +71,8 @@
 which the accounted is locked.
 The date may also be expressed in the format YYYY-MM-DD (or the format more
 commonly used in your area).
+Passing the number -1 as the \fIexpiredate\fR will remove an accounts
+expiration date.
 A user whose account is locked must contact the system administrator before
 being able to use the system again.
 .PP
